console/src/components/Services/Data/TableModify/ModifyActions.js

```diff
@@ -128,6 +128,57 @@ const resetPrimaryKeys = () => ({
   type: RESET_PRIMARY_KEY,
 });
 
+export const removeCheckConstraint = constraintName => (dispatch, getState) => {
+  const isOk = window.confirm('Are you absolutely sure?');
+  if (!isOk) return;
+
+  const { currentTable: tableName, currentSchema } = getState().tables;
+
+  const tableSchema = getState().tables.allSchemas.find(s => {
+    return s.table_name === tableName && s.table_schema === currentSchema;
+  });
+
+  const constraint = tableSchema.check_constraints.find(
```

Comment:
create util functions for this is `pgUtils.js` called  `getTableCheckConstraints(table)` and `getTableCheckConstraint(table, constraintName)`. 
Rule of thumb we do not deconstruct PG objects outside this file and we try to use the base fns to define functions on top of them. eg. `getTableConstraint` should use `getTableConstraints` internally. You can see this pattern being followed in the pgutils file already to avoid duplication of logic